consent-ws/src/test/java/org/broadinstitute/consent/http/ConsentResourceTest.java

```diff
@@ -143,4 +159,103 @@ private Election updateElection(String consentId, Integer electionId) throws IOE
         return updated;
     }
 
+    @Test
+    public void testUpdateConsentGroupName() throws Exception {
+        Client client = ClientBuilder.newBuilder().register(MultiPartFeature.class).build();
```

Comment:
Do you really need `MultiPartFeature`? Tests seem to run fine without that - I think it was necessary for file uploading.